lib/g_searcher_web/validators/report.ex

```diff
@@ -0,0 +1,26 @@
+defmodule GSearcherWeb.Validators.Report do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  embedded_schema do
+    field :title, :string
+    field :csv, :any, virtual: true
+  end
+
+  def changeset(data \\ %__MODULE__{}, params) do
+    data
+    |> cast(params, [
+      :title,
+      :csv
+    ])
+    |> validate_required([:title, :csv])
+    |> validate_is_csv()
```

Comment:
just naming `validate_valid_csv`, what do you think?

Reply:
I think maybe calling it something like `validate_csv_type` makes more sense since we're not checking the internals of the CSV that could make it invalid like improper formatting. 🤔